dbt/adapters/spark/connections.py

```diff
@@ -138,6 +138,16 @@ def __post_init__(self):
                     "`pip install dbt-spark[session]`\n\n"
                     f"ImportError({e.msg})"
                 ) from e
+        
+        if self.method != SparkConnectionMethod.SESSION:
+            # Remove https:// if it is added,
+            # allowing users to provide the host name with or without https://
+            if self.host.startswith("https://"):
+                self.host = self.host.replace("https://", "")
```

Comment:
@tim-steinkuhler : Could you rewrite this to?  A little safer than the replace

```suggestion
                self.host = self.host[len("https://"):]
```

Reply:
Thanks for having a look @JCZuurmond! I changed it to `self.host = self.host.lstrip("https://")`